resources/assets/uad_lists.json

```
@@ -34356,5 +34356,269 @@
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.android.compos.payload": {
+    "list": "Oem",
+    "description": "CompOS\nHas something to 'com.android.compos'.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  "com.android.devicelockcontroller": {
+    "list": "Aosp",
+    "description": "This app can't be uninstalled or disabled.\nCan restrict this device if the owner doesn't make payments per month for the new phone.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Unsafe"
+  },
+  "com.vos.user.vit": {
+    "list": "Oem",
+    "description": "App for testing things.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.vos.as.vit": {
+    "list": "Oem",
+    "description": "App for testing things.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.vivo.game": {
+    "list": "Oem",
+    "description": "Game Space. Not sure if its useful for gaming.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.vivo.globalanimation.resources": {
+    "list": "Oem",
+    "description": "GlobalanmationResources\nHas animation booting probably, safe to remove.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.vivo.imanager": {
+    "list": "Oem",
+    "description": "iManager\nA vivo app consists of Phone Management includes Space Cleanup, Security Scan, Data Traffic Management, Apps & Notifications & Utility Tools includes App encryption, Battery Management, App Clone, Network Management etc.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  "com.mediatek.magtapp": {
+    "list": "Misc",
+    "description": "MAGTApp\nUnknown, has GameEventService, MAGTEventAppReceiver, but code not looks very useful.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.vivo.monochromeicons": {
+    "list": "Oem",
+    "description": "MonochromeIcons\nIn code found something to vivo icons.\nIn Android 14.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.vivo.livewallpaper.ocean": {
+    "list": "Oem",
+    "description": "Live Wallpaper Ocean.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.vivo.livewallpaper.oceanvertical": {
+    "list": "Oem",
+    "description": "Live Wallpaper Oceanvertical.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.vivo.simplelauncher": {
+    "list": "Oem",
+    "description": "Simple launcher, Simple View.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```
You need a replacement installed first I think.

---

its a optional feature